packages/core-components/src/components/Select/Select.tsx

```diff
@@ -203,7 +205,7 @@ export function SelectComponent(props: SelectProps) {
           value={value}
           native={native}
           disabled={disabled}
-          data-testid="select"
+          data-testid={testid}
```

Comment:
Could you please help us add a test for this new property? The test will fail if someone accidentally remove the property in the future 🙏🏻